### PR TITLE
[devtools] run chromium with devtools features via puppeteer

### PR DESCRIPTION
**Pull request by kaycebasques**
@ebidel WDYT of this little post showing how to use puppeteer to enable devtools features? It's intentionally minimal. If there's a lot of interest or confusion, I can follow up with more detailed guidance later.

@petele I added a "puppeteer" tag to the post. Doesn't look like have that tag yet. Not sure how to do that. update: I removed it

What's changed, or what was fixed?
- adds "devtools without devtools" update

**Fixes:** N/A

**Target Live Date:** 2018-01-22

- [ ] This has been reviewed and approved by @ebidel 
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2018/01/devtools-without-devtools.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Use Puppeteer to launch Chromium with DevTools features enabled.
+
+{# wf_updated_on: 2018-01-22 #}
+{# wf_published_on: 2018-01-22 #}
+{# wf_tags: devtools,puppeteer #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Use Puppeteer to launch Chromium with DevTools features enabled. #}
+{# wf_blink_components: Platform>DevTools #}
```
Comment (ebidel) [inline src/content/en/updates/2018/01/devtools-without-devtools.md]: also Internals>Headless

File: src/content/en/updates/2018/01/devtools-without-devtools.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    const puppeteer = require('puppeteer'); // version 1.0.0
+    
+    (async () => {
+      // Prevent Puppeteer from showing the "Chrome is being controlled by automated test
+      // software" prompt, but otherwise use Puppeteer's default args.
+      const args = puppeteer.defaultArgs().filter(flag => flag !== '--enable-automation');
+      const browser = await puppeteer.launch({
+        headless: false,
+        ignoreDefaultArgs: true,
+        args: args
```
Comment (ebidel) [inline src/content/en/updates/2018/01/devtools-without-devtools.md]: you dont need the val if the var name is the same as the key

`args: args` -> `args,`

**Comment by kaycebasques** [comment]
I want to keep it small, to pique people's curiousty. If we get a lot of interest, I can follow up with an in-depth post.

However, I'll update the post to explain that there's lots more that you can do, to solicit community contributions, etc.

File: src/content/en/updates/2018/01/devtools-without-devtools.md (hunk 2 above)
Comment (kaycebasques) [reply src/content/en/updates/2018/01/devtools-without-devtools.md]: TIL, thanks

File: src/content/en/updates/2018/01/devtools-without-devtools.md (hunk 1 above)
Comment (kaycebasques) [reply src/content/en/updates/2018/01/devtools-without-devtools.md]: Noice

**Comment by kaycebasques** [comment]
Looks good enough. Gonna merge.

**Pull request merged**